build.gradle

```
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+    id 'me.champeau.jmh' version '0.6.8' // Подключение плагина JMH
+}
+
+group = 'ru.highloadjava.concurrentlab'
+version = '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.openjdk.jmh:jmh-core:1.36'
+    annotationProcessor 'org.openjdk.jmh:jmh-generator-annprocess:1.36'
+    implementation 'io.qala.datagen:qala-datagen-junit5:2.5.0'
```

```suggestion
    testImplementation 'io.qala.datagen:qala-datagen-junit5:2.5.0'
```

Ну и переместить пониже, чтобы оно осознанно было